californium-core/src/main/java/org/eclipse/californium/core/coap/ClientObserveRelation.java

```diff
@@ -136,6 +145,26 @@ public ClientObserveRelation(Request request, Endpoint endpoint, ScheduledThread
 		this.request.setProtectFromOffload();
 	}
 
+	/**
+	 * Checks if the relation is reregistered and cancelled confirmable or
+	 * non-confirmable.
+	 *
+	 * @return true, if the relation is registered and cancelled confirmable
+	 */
+	public boolean isConfirmable() {
+		return confirmable;
+	}
+
+	/**
+	 * Set the relation register and cancel message type.
+	 *
+	 * @param confirmable when {@code True} register and cancel messages are
+	 *            sent confirmable, non-confirmable otherwise.
+	 */
```

Comment:
I would prefer `true` for javadoc.

Please add a 

```
@since 3.8
```

Reply:
OK